tests/e2e/interOp/validation_of_operating_modes/bridge_mode/country_code_channel_division/country_code_GB/iOS/test_20mhz.py

```diff
@@ -0,0 +1,1773 @@
+    @pytest.mark.twog
+    @pytest.mark.wpa2_personal
+    @pytest.mark.twentyMhz
+    @pytest.mark.firstrun
+    def test_ClientConnect_bridge_wpa2_chn1_20Mhz_GB_2g(self, request, get_vif_state, get_ap_logs,
+                                                         get_ToggleAirplaneMode_data, setup_perfectoMobile_iOS):
+        profile_data = setup_params_general1["ssid_modes"]["wpa2_personal"][0]
+        ssidName = profile_data["ssid_name"]
+        ssidPassword = profile_data["security_key"]
+        print("SSID_NAME: " + ssidName)
+        print("SSID_PASS: " + ssidPassword)
+        get_vif_state.append(ssidName)
+        if ssidName not in get_vif_state:
+            allure.attach(name="retest,vif state ssid not available:", body=str(get_vif_state))
+            pytest.xfail("SSID NOT AVAILABLE IN VIF STATE")
+
+        report = setup_perfectoMobile_iOS[1]
+        driver = setup_perfectoMobile_iOS[0]
+        connData = get_ToggleAirplaneMode_data
+
+        # Set Wifi/AP Mode
+        ip, is_internet = get_ip_address_ios(request, ssidName, ssidPassword, setup_perfectoMobile_iOS, connData)
+
+        if ip:
+            if is_internet:
+                text_body = ("connected to " + ssidName + " (" + ip + ") " + "with internet")
+            else:
+                text_body = ("connected to " + ssidName + " (" + ip + ") " + "without internet")
+            print(text_body)
+            allure.attach(name="Connection Status: ", body=str(text_body))
+            assert True
+        else:
+            allure.attach(name="Connection Status: ", body=str("Device is Unable to connect"))
+            assert False
+
+
+setup_params_general2 = {
+    "mode": "BRIDGE",
+    "ssid_modes": {
+        "wpa2_personal": [{"ssid_name": "ssid_wpa2_2g", "appliedRadios": ["2G"], "security_key": "something"},
+                          {"ssid_name": "ssid_wpa2_5g", "appliedRadios": ["5G"], "security_key": "something"}
+                          ]},
+    "rf": {"5G":
+        {'band': '5G',
+        'country': 'GB',
+        "channel-mode": "VHT",
+        'channel-width': 20,
+        "channel": 40},
+        "2G":
+        {'band': '2G',
+        'country': 'GB',
+        "channel-mode": "VHT",
+        'channel-width': 20,
+        "channel": 2}
+           },
+    "radius": False
+}
+
+for sec_modes in setup_params_general2['ssid_modes'].keys():
+    for i in range(len(setup_params_general2['ssid_modes'][sec_modes])):
+        N = 3
+        rand_string = (''.join(random.choices(string.ascii_uppercase +
+                                     string.digits, k=N)))+str(int(time.time_ns())%10000)
+        setup_params_general2['ssid_modes'][sec_modes][i]['ssid_name'] = setup_params_general2['ssid_modes'][sec_modes][i]['ssid_name'] + "_"+ rand_string
+
+# @allure.suite(suite_name="interop country code")
+# @allure.sub_suite(sub_suite_name="Bridge Mode country code")
+# @pytest.mark.InteropsuiteA
+# @allure.feature("BRIDGE MODE CLIENT CONNECT")
```

Comment:
Don't comment these lines and remove the marker in them
```suggestion
@allure.suite(suite_name="interop country code")
@allure.sub_suite(sub_suite_name="Bridge Mode country code (GB)")
@allure.feature("BRIDGE MODE CLIENT CONNECT WITH COUNTRY CODE")
```